lib/constants.dart

```diff
@@ -299,4 +299,11 @@ class PTOrientation {
   static const reversePortrait = 9;
   static const userLandscape = 11; // Only changes direction if user has enabled sensor-based rotation.
   static const userPortrait = 12; // Only changes direction if user has enabled sensor-based rotation.
+}
+
+// ViewModePickerItem defines the items in the view mode dialog.
+class ViewModePickerItem {
+  static const Crop = "viewModeCrop";
+  static const Rotation = "viewModeRotation";
+  static const ColorMode = "viewModeColorMode";
```

Comment:
What if a customer wants to hide "sepia" only? I think we should treat each row here separately, not as a group.

Reply:
hiding sepia only is not supported on android, that was not stated as requirement in the channel